clang/lib/Driver/ToolChains/Clang.cpp

```diff
@@ -9119,7 +9119,8 @@ void SPIRVTranslator::ConstructJob(Compilation &C, const JobAction &JA,
     std::string DefaultExtArg =
         ",+SPV_EXT_shader_atomic_float_add,+SPV_EXT_shader_atomic_float_min_max"
         ",+SPV_KHR_no_integer_wrap_decoration,+SPV_KHR_float_controls"
-        ",+SPV_KHR_expect_assume,+SPV_KHR_linkonce_odr";
+        ",+SPV_KHR_expect_assume,+SPV_KHR_linkonce_odr"
+        ",+SPV_KHR_uniform_group_instructions";
```

Comment:
Let's enable the extension on any target but FPGA H/W aka move this code to line 9154.